test/qt3testsBenchmark.ts

```diff
@@ -0,0 +1,274 @@
+import { Suite } from 'benchmark';
+
+import {
+	evaluateXPathToBoolean,
+	evaluateXPathToFirstNode,
+	evaluateXPathToNodes,
+	evaluateXPathToNumber,
+	evaluateXPathToString,
+	Language,
+	Document,
+} from 'fontoxpath';
+
+import testFs from 'test-helpers/testFs';
+import { ALL_TESTS_QUERY, getAllTestSets, getArguments, getFile } from 'test-helpers/qt3TestsTools';
+import getPerformanceTests from 'test-helpers/getPerformanceTests';
+import { Node, Element } from 'slimdom';
+
+const suite: Suite = new Suite();
+
+function addTests(functions) {
+	Object.keys(functions).forEach((name) => {
+		suite.add(name, functions[name]);
+	});
+}
+
+function runTests() {
+	console.log('Tests are running...');
+	suite
+		// add listeners
+		.on('cycle', (event) => {
+			console.log(String(event.target));
+		})
+		.on('complete', () => {
+			testFs.writeFileSync(
```

Comment:
Why do we write the results to file? I thought we'd just write these to the console output.

Reply:
discussed and done